beginner_source/text_sentiment_ngrams_tutorial.py

```diff
@@ -2,7 +2,7 @@
 Text classification with the torchtext library
 ==================================
 
-In this tutorial, we will show how to use the new torchtext library to build the dataset for the text classification analysis. In the nightly release of the torchtext library, we provide a few prototype building blocks for data processing. Users will have the flexibility to
+In this tutorial, we will show how to use the new torchtext library to build the dataset for the text classification analysis. Users will have the flexibility to
```

Comment:
We don't need to say "new" torchtext library anymore, because the datasets are now part of the top folder.

Reply:
Fixed.